src/mcl_3dl.cpp

```diff
@@ -135,6 +135,7 @@ class MCL3dlNode
     }
 
     pc_map_.reset(new pcl::PointCloud<PointType>);
+    pcl_conversions::toPCL(ros::Time::now(), pc_map_->header.stamp);
```

Comment:
Why it isn't
```suggestion
    pcl_conversions::toPCL(msg->header.stamp, pc_map_->header.stamp);
```
?

Reply:
Fixed to use msg->header.stamp when it is valid.